charts/composer/values.yaml

```diff
@@ -19,7 +19,8 @@ config:
   maxSubmitInterval: 2000
   sequencerAddressPrefix: astria
   sequencerNativeAssetBaseDenomination: "nria"
-  sequencerRpc: ""
+  cometbftEndpoint: "http://node0-sequencer-rpc-service.astria-dev-cluster.svc.cluster.local:26657"
```

Comment:
Should template from `evm-stack` chart, set value to `""` in `composer` chart, use global values of the `evm-stack` to override, similar to other values.